### PR TITLE
ticdc: cdc is GA since v4.0.6

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? (Required)

CDC is GA since v4.0.6, update related docs

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @TomShawn @leoppro

**Review by leoppro — COMMENTED** [review]
LGTM

File: ticdc-deployment-topology.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ aliases: ['/docs-cn/dev/ticdc-deployment-topology/','/docs-cn/dev/reference/tool
 
 > **注意：**
 >
-> TiCDC 目前为实验特性，不建议在生产环境中使用。
+> TiCDC 从 v4.0.6 成为正式功能，可用于生产环境。
```
Comment (TomShawn) [inline ticdc-deployment-topology.md:9]:
```suggestion
> TiCDC 从 v4.0.6 起成为正式功能，可用于生产环境。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**